python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Commands.xml

```diff
@@ -16,24 +16,24 @@
   <SALCommand>
     <Subsystem>Electrometer</Subsystem>
     <EFDB_Topic>Electrometer_command_setDigitalFilter</EFDB_Topic>
-    <Description>Configure the digital filter inside the Electrometer controller. Be aware that activating any filters will reduce the number of readings per second the device will be able to handle</Description>
+    <Description>Configures the digital filter in the device. Filtering stabilizes noisy measurements caused by noisy input signals. Be aware that activating any filters will reduce the number of readings per second the device will be able to handle. The Electrometer has two types of filters: Averaging and Median. Note that both filters can be enabled at the same time. If so, each reading will reflect the median for the average readings. In order to enable either of these filters, one must set the activateFilter to ON (1). Any reading taken when a filter is enabled will be the result of the filtering process.</Description>
     <item>
       <EFDB_Name>activateFilter</EFDB_Name>
-      <Description>Activate the filter. If this is OFF (set to 0), none of the filters will operate regardless of their settings below</Description>
+      <Description>Enables ability to the activate either filter. If this is OFF (set to 0), neither of the filters will operate regardless of their settings below.</Description>
       <IDL_Type>boolean</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>activateAvgFilter</EFDB_Name>
-      <Description>Activate the average filter inside the electrometer, it uses the Default value inside the device (last 10 readings) to do the average calculation, 1 for ON and 0 for OFF </Description>
+      <Description>Activate the average filter (scalar type) inside the electrometer, 1 for ON and 0 for OFF. This filter takes the average of the last 10 readings (default) in the moving mode (first-in, first-out stack).</Description>
```

Comment:
I took this from the user guide close to a year ago. At the time I didn't know what it meant but thought it might be important. Since I still don't know what it means, I'll remove it.